README.md

```diff
@@ -1,117 +1,200 @@
+    publicassets/87dd2e5a-42b4-49a5-8693-97f40a5af7f8/events/a022f458-8e55-4d63-a200-4172a42fc2aa
     leaf hash: bfc511ab1b880b24bb2358e07472e3383cdeddfbc4de9d66d652197dfb2b6633
     OK|663 334|[aea799fb2a8..., proof path nodes, ...f0a52d2256c235]
+    ```
 
+**Note:** _To minimize veracity output, remove `--loglevel`, checking the exit code of 0 (`echo $?`) for a successful verification._
 
 The elided proof path at time of writing was:
 
-    [aea799fb2a8c4bbb6eda1dd2c1e69f8807b9b06deeaf51b9e0287492cefd8e4c, 9f0183c7f79fd81966e104520af0f90c8447f1a73d4e38e7f2f23a0602ceb617, da21cb383d63896a9811f06ebd2094921581d8eb72f7fbef566b730958dc35f1, 51ea08fd02da3633b72ef0b09d8ba4209db1092d22367ef565f35e0afd4b0fc3, 185a9d55cf507ef85bd264f4db7228e225032c48da689aa8597e11059f45ab30, bab40107f7d7bebfe30c9cea4772f9eb3115cae1f801adab318f90fcdc204bdc, 94ca607094ead6fcd23f52851c8cdd8c6f0e2abde20dca19ba5abc8aff70d0d1, ba6d0fd8922342aafbba6073c5510103b077a7de9cb2d72fb652510110250f9e, 7fafc7edc434225afffc19b0582efa2a71b06a2d035358356df0a52d2256c235, b737375d837e67ee7bce182377304e889187ef0f335952174cb5bf707a0b4788]
+```output
+[aea799fb2a8c4bbb6eda1dd2c1e69f8807b9b06deeaf51b9e0287492cefd8e4c,
+9f0183c7f79fd81966e104520af0f90c8447f1a73d4e38e7f2f23a0602ceb617, 
+a21cb383d63896a9811f06ebd2094921581d8eb72f7fbef566b730958dc35f1, 
+1ea08fd02da3633b72ef0b09d8ba4209db1092d22367ef565f35e0afd4b0fc3, 
+85a9d55cf507ef85bd264f4db7228e225032c48da689aa8597e11059f45ab30, 
+ab40107f7d7bebfe30c9cea4772f9eb3115cae1f801adab318f90fcdc204bdc, 
+4ca607094ead6fcd23f52851c8cdd8c6f0e2abde20dca19ba5abc8aff70d0d1, 
+a6d0fd8922342aafbba6073c5510103b077a7de9cb2d72fb652510110250f9e, 
+fafc7edc434225afffc19b0582efa2a71b06a2d035358356df0a52d2256c235, 
+737375d837e67ee7bce182377304e889187ef0f335952174cb5bf707a0b4788]
+```
+
+## Verify Tamper Resiliency
 
-The same command accepts the result of a DataTrails list events call, e.g.
+One of the many scenarios DataTrails prevents is tampering if and when information was written to the ledger.
 
-    DATATRAILS_URL=https://app.datatrails.ai
-    PUBLIC_TENANT_ID=tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
-    PUBLIC_ASSET_ID=publicassets/87dd2e5a-42b4-49a5-8693-97f40a5af7f8
+1. To simulate backdating, the following backdates one of the events in the log:
+
+    ```console
+    sed -i -e 's/2024-05-24T07:27:00.200Z/2024-04-24T07:27:00.200Z/g' ./event.json
+    ```
+
+1. Re-verify inclusion with `veracity verify-included`, noting the error
+
+    ```console
+    cat event.json | \
+        veracity --data-url $DATATRAILS_URL/verifiabledata \
+        --tenant=$PUBLIC_TENANT_ID \
+        --loglevel=INFO \
+        verify-included
+    ```
+
+1. View the output
 
+    ```output
+    ...
+    error: the entry is not in the log. for tenant tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
+    ```
+
+## Verify All Events
+
+The `veracity verify-included` command accepts the result of a DataTrails list events call
```

Comment:
suggestion

```
. This verifies inclusion of each single even in the returned list
```

Because Joe pointed out the potential confusion versus what the completeness command will do

Reply:
see #22